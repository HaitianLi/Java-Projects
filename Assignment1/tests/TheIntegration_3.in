add collection books.csv
add collection books2.csv
list all
list GENRES
List Authors
add member jack
add member mage
rent 100000 211111
rent 100001 211121
list available
number copies
relinquish 100000 211111
relinquish 100001 211121
common 100000 100001
exit