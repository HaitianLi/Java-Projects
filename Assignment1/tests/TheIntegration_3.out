user: 17 books successfully added.

user: 19 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
come second with serial 111234 (Haitian Li)
third with serial 111456 (Haitian Li)
should come first 111144 (Haitian Li)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
final with 211204 (Bob Kane)
third show with 211203 (J.K. Rowling)
second show with 211200 (J.K. Rowling)
first show with 211120 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Comedy
Comic
Epic
Fantasy
Genre testcase
Historical Fiction
Horror
Poetry
Test Case
Test Case 2
War

user: Bob Kane
Douglas Adams
Haitian Li
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: Success.

user: Success.

user: No such book in system.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)
come second with serial 111234 (Haitian Li)
third with serial 111456 (Haitian Li)
should come first 111144 (Haitian Li)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
final with 211204 (Bob Kane)
third show with 211203 (J.K. Rowling)
second show with 211200 (J.K. Rowling)
first show with 211120 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)
Wasteland (T.S. Eliot)
Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Aeneid (Virgil): 2
Batman Volume 1 (Bob Kane): 1
Biggles in the Battle (W.E. Johns): 2
Halloween (Jamie Lee Curtis): 2
Harry Potter 1 (J.K. Rowling): 1
Harry Potter 2 (J.K. Rowling): 1
Harry Potter 3 (J.K. Rowling): 1
Harry Potter 4 (J.K. Rowling): 2
Harry Potter 5 (J.K. Rowling): 2
Harry Potter 6 (J.K. Rowling): 2
So Long and Thanks for all the Fish (Douglas Adams): 2
The Hitchhiker's Guide to the Galaxy (Douglas Adams): 2
The Hobbit (J.R.R. Tolkien): 2
The Iliad (Homer): 2
The Odyssey (Homer): 2
To Kill a Mockingbird (Harper Lee): 1
Wasteland (T.S. Eliot): 2
come second with serial 111234 (Haitian Li): 1
final with 211204 (Bob Kane): 1
first show with 211120 (J.K. Rowling): 1
second show with 211200 (J.K. Rowling): 1
should come first 111144 (Haitian Li): 1
third show with 211203 (J.K. Rowling): 1
third with serial 111456 (Haitian Li): 1

user: No such book in system.

user: Success.

user: No common books.

user: Ending Library process.
